alpine-chroot-install

```diff
@@ -345,6 +362,9 @@ printf '%s\n' \
 dump_alpine_keys etc/apk/keys/
 
 cp /etc/resolv.conf etc/resolv.conf
+if [ "x$FORCE_DNS" != "x" ]; then
```

Comment:
Why the heck people still do this? It’s a workaround for a bug in one particular shell implementation that was already fixed many decades ago! Nowadays you can find this buggy implementation maybe in some museum of computing, if at all.